test/e2e_node/podresources_test.go

```diff
@@ -279,9 +306,62 @@ var _ = SIGDescribe("POD Resources [Serial] [Feature:PODResources][NodeFeature:P
 			framework.ExpectNoError(err)
 
 			cli, conn, err := podresources.GetV1Client(endpoint, defaultPodResourcesTimeout, defaultPodResourcesMaxSize)
+			framework.ExpectNoError(err)
 			defer conn.Close()
 
 			podresourcesListTests(f, cli, sd)
 		})
+
+		ginkgo.It("should return the expected device count", func() {
+			// this is a very rough check. We just want to rule out system that does NOT have any SRIOV device.
+			if sriovdevCount, err := countSRIOVDevices(); err != nil || sriovdevCount == 0 {
```

Comment:
Do we have any machine in the test infrastructure that has SRIOV devices? If so how and when are tests ran, if not what is our plan for continuous integration?

Reply:
at this moment, no. This is a problem for few other e2e tests. There is an ongoing conversation - which I need to push forward more - about if and how we can add these lanes to the k8s ci.
I have few internal jenkins instances inside RH on which we continuously run the tests. Another option is somehow hooking these instances to k8s prow, but it's challenging as well.

The problem is to get any device supported by device plugins, sriov happens to be the easiest for me to get access to but I'm very open to use any other device we can use.
The only requirement is to be a real device, no fakes/mocks (maybe we can re-evaluate this requirement?)